docs/Operators.md

```diff
@@ -2948,7 +2948,7 @@ Other versions of this operator: <a href="Changelog.md#ConvTranspose-1">ConvTran
 <dt><tt>kernel_shape</tt> : list of ints</dt>
 <dd>The shape of the convolution kernel. If not present, should be inferred from input W.</dd>
 <dt><tt>output_padding</tt> : list of ints</dt>
-<dd>The zero-padding added to one side of the output. This is also called adjs/adjustment in some frameworks.</dd>
+<dd>Additinal elements added to the side with higher coordinate indexes in the output. Each element in "output_padding" cannot be larger than the stride along the associated axis. By default, this attribute is a zero vector. Note that this attribute doesn't directly affect the computed output values. It only controls the selection of the computed values. This is also called adjs or adjustment in some frameworks.</dd>
```

Comment:
Q: is "selection of the computed values" equivalent to saying that it affects only the shape of the output?

Reply:
Yes.